packages/services/src/kernel/default.ts

```diff
@@ -225,6 +240,10 @@ export class KernelConnection implements Kernel.IKernelConnection {
     return this._specPromise;
   }
 
+  supportsSubshells(): boolean {
```

Comment:
Maybe we can add a quick docstring here for consistency with the rest of the file?